bip-taproot.mediawiki

```diff
@@ -236,6 +236,22 @@ def taproot_output_script(internal_pubkey, script_tree):
 
 [[File:bip-taproot/tree.png|frame|This diagram shows the hashing structure to obtain the tweak from an internal key ''P'' and a Merkle tree consisting of 5 script leaves. ''A'', ''B'', ''C'' and ''E'' are ''TapLeaf'' hashes similar to ''D'' and ''AB'' is a ''TapBranch'' hash. Note that when ''CDE'' is computed ''E'' is hashed first because ''E'' is less than ''CD''.]]
 
+To spend this output using script ''D'', the control block would contain the following data in this order:
+
+<source>
+[Leaf version of D + control bits][internal key P][C][E][AB]
+</source>
+
+The TapTweak would then be computed as described in [[#script-validation-rules]] like so:
+
+<source>
+D = tagged_hash("TapLeaf", bytes([leaf_version]) + ser_script(script))
```

Comment:
Better use `||` for concatenation as in the rest of the document. (EDIT: oh nvm, this is supposed to be python code).

Reply:
:+1: